File: January/Day3/leetcode_944.java
class Solution {
    public int minDeletionSize(String[] strs) {
        int count=0;
        for(int i=0;i<strs[0].length();i++){
            for(int j=1;j<strs.length;j++){
                if(strs[j].charAt(i)<strs[j-1].charAt(i)){
                    count++;
                    break;
                }
            }
        }
        
        return count;
    }
}

File: January/Day3/leetcode_944.cpp
class Solution {
public:
    int minDeletionSize(vector<string>& strs) {

      int ans = 0;

          for(int j = 0; j<strs[0].size(); j++)
          {
            for(int i = 1; i<strs.size(); i++)
            {
            if(strs[i-1][j]>strs[i][j])
            {
                ans++;
                break;
            }  
          }
      }
      return ans;

    }
};
